contrib/util/check-config.sh

```diff
@@ -81,6 +84,11 @@ color() {
   printf '\033['"$codes"'m'
 }
 wrap_color() {
+  if [ $RAW_OUTPUT -eq 1 ]; then
```

Comment:
this seems fine, but an even more minimal change might be to make `color()` return early if this env var is set?

Also, I think [`NO_COLOR`](https://www.gnu.org/software/gettext/libtextstyle/manual/html_node/The-NO_005fCOLOR-variable.html) is a more widely supported variable to use for this. If it is set to anything other than an empty string, color output should be suppressed.